docs/source/conf.py

```diff
@@ -10,9 +10,9 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../vayesta/'))
```

Comment:
Is this just relative to the Makefile? Maybe better as `os.path.abspath(os.path.join(__file__, "..", "..", "..", "vayesta"))`?

Reply:
This is not  for Makefile, this is where sphinx-apidocs is looking for te module list. This is the way has been suggested in forums:

https://stackoverflow.com/questions/17086273/specifying-a-relative-path-in-sphinx-conf-py-for-collaborative-documentation